packages/repository-modernizer/executors/config.yaml

```diff
@@ -1,35 +1,35 @@
 repositoryModernizer:
     # groupId to be used for newly created packages
-    groupId:
+    groupId: com.ugh.apps.aem
```

Comment:
@sarvkuma please remove the configs, we need to keep the config clean in public repo.